oolab/src/main/java/agh/ics/oop/GrassField.java

```diff
@@ -0,0 +1,90 @@
+package agh.ics.oop;
+
+import java.util.Random;
+
+import static java.lang.Math.round;
+import static java.lang.Math.sqrt;
+
+public class GrassField extends AbstractWorldMap {
+    private int fields;
+
+    private final Random randomGenerator;
+
+    public GrassField(int fields) {
+        this.fields = fields;
+        randomGenerator = new Random();
+        placeFields(fields);
```

Comment:
tak jak na poprzedniej laborce - mniejszy konstruktor jest szczególnym przypadkiem większego, tylko trzeba by dodać trzeci (może być prywatny), który zamiast seed przyjmie całe `Random`, wtedy to się poskłada.